docs/index.rst

```diff
@@ -10,12 +10,27 @@ ops module
 ==========
 
 .. automodule:: ops
+   :exclude-members: main
 
 
-main
------------
+ops.main entry point
+====================
+.. autofunction:: ops.main
+   :noindex:
+
+
+_main module
+------------
+
+.. automodule:: ops._main
+   :noindex:
```

Comment:
I think we probably shouldn't expose this in the docs. Having the legacy one there to point out the deprecation is good, though.

Reply:
done

Reply:
So I've hidden the module, but that also means that all of the module content, incl. `_Dispatcher` is gone from the docs.

Is that acceptable/intended, or do we want `_main` content documented, but not the `_main.main()` function?

Reply:
We currently don't have any of the private methods/classes in the docs, so that should be fine. Most of them are undocumented anyway, and I don't think we want to be exposing them to people.